packages/utilities/README.md

```diff
@@ -13,6 +13,84 @@
+        -   [`filterNullAndUndefined`](#filternullandundefined)
+        -   [`filterNullAndUndefinedAndEmpty`](#filternullandundefinedandempty)
+        -   [`filterNullAndUndefinedAndZero`](#filternullandundefinedandzero)
+        -   [`getDeepObjectKeys`](#getdeepobjectkeys)
+        -   [`hasAtLeastOneKeyInMap`](#hasatleastonekeyinmap)
+        -   [`inlineCodeBlock`](#inlinecodeblock)
+        -   [`isClass`](#isclass)
+        -   [`isFunction`](#isfunction)
+        -   [`isNullOrUndefined`](#isnullorundefined)
+        -   [`isNullOrUndefinedOrEmpty`](#isnullorundefinedorempty)
+        -   [`isNullOrUndefinedOrZero`](#isnullorundefinedorzero)
+        -   [`isNumber`](#isnumber)
+        -   [`isObject`](#isobject)
+        -   [`isPrimitive`](#isprimitive)
+        -   [`isThenable`](#isthenable)
+        -   [`lazy`](#lazy)
+        -   [`makeObject`](#makeObject)
+        -   [`mergeDefault`](#mergedefault)
+        -   [`mergeObjects`](#mergeobjects)
+        -   [`noop`](#noop)
+        -   [`objectToTuples`](#objecttotuples)
+        -   [`partition`](#partition)
+        -   [`pickRandom`](#pickRandom)
+        -   [`range`](#range)
+        -   [`regExpEsc`](#regexesc)
+        -   [`roundNumber`](#roundnumber)
+        -   [`sleep`](#sleep)
+        -   [`splitText`](#splitText)
+        -   [`throttle`](#throttle)
+        -   [`toTitleCase`](#totitlecase)
+        -   [`tryParseJSON`](#tryparsejson)
+        -   [`tryParseURL`](#tryparseurl)
+    -   [Typescript Utilities](#typescript-utilities)
+        -   [Functions](#functions)
+            -   [`cast`](#cast)
+            -   [`objectEntries`](#objectentries)
+            -   [`objectKeys`](#objectkeys)
+            -   [`objectValues`](#objectvalues)
+        -   [Types](#types)
+            -   [`Primitive`](#primitive)
+            -   [`Builtin`](#builtin)
+            -   [`DeepRequired`](#deeprequired)
+            -   [`RequiredExcept`](#requiredexcept)
+            -   [`PartialRequired`](#partialrequired)
+            -   [`DeepPartial`](#deeppartial)
+            -   [`ArgumentTypes`](#argumenttypes)
+            -   [`Arr`](#arr)
+            -   [`Ctor`](#ctor)
+            -   [`AbstractCtor`](#abstractctor)
+            -   [`Constructor`](#constructor)
+            -   [`AbstractConstructor`](#abstractconstructor)
+            -   [`FirstArgument`](#firstargument)
+            -   [`SecondArgument`](#secondargument)
+            -   [`Awaitable`](#awaitable)
+            -   [`Nullish`](#nullish)
+            -   [`NonNullableProperties`](#nonnullableproperties)
+            -   [`NonNullObject`](#nonnullobject)
+            -   [`AnyObject`](#anyobject)
+            -   [`PickByValue`](#pickbyvalue)
+            -   [`Mutable`](#mutable)
+            -   [`StrictRequired`](#strictrequired)
+            -   [`ArrayElementType`](#arrayelementtype)
+-   [Buy us some doughnuts](#buy-us-some-doughnuts)
+-   [Contributors ✨](#contributors-%E2%9C%A8)
```

Comment:
```suggestion
-   [Contributors ✨](#contributors-)
```

Something to be fixed elsewhere too but otherwise this link is broken :(

Reply:
Will your change fix it or is it just a placeholder?